pom.xml

```diff
@@ -43,6 +43,7 @@
     <properties>
         <check.fail-spotbugs>true</check.fail-spotbugs>
         <check.spotbugs-exclude-filter-file>spotbugs-exclude.xml</check.spotbugs-exclude-filter-file>
+        <killbill-api.version>0.54.0-19db89f-SNAPSHOT</killbill-api.version>
     </properties>
```

Comment:
The latest version of [killbill-api](https://github.com/killbill/killbill-api) seems to be `0.54.0-01c38a9-SNAPSHOT` based on the [commits](https://github.com/killbill/killbill-api/commits/work-for-release-0.23.x). Also, the [killbill pom](https://github.com/killbill/killbill/blob/344816ee793a967212979e32292301db810f5ee4/pom.xml#L64) seems to be specifying this version.

Reply:
Hmm.. according to published artifact on sonatype, there's 3 [killbill-api](https://oss.sonatype.org/content/repositories/snapshots/org/kill-bill/billing/killbill-api/) that released on November 23:
- [`0.54.0-01c38a9-SNAPSHOT`](https://oss.sonatype.org/content/repositories/snapshots/org/kill-bill/billing/killbill-api/0.54.0-01c38a9-SNAPSHOT),
- [`0.54.0-0c033a7-SNAPSHOT`](https://oss.sonatype.org/content/repositories/snapshots/org/kill-bill/billing/killbill-api/0.54.0-0c033a7-SNAPSHOT), and
- [`0.54.0-19db89f-SNAPSHOT`](https://oss.sonatype.org/content/repositories/snapshots/org/kill-bill/billing/killbill-api/0.54.0-19db89f-SNAPSHOT)

`19db89f` wins because released ~10 minutes later :smile: . 

TBH I don't know why this happened, and I found something like this a few times, but I simply pick `19db89f` because it is the latest. I'll set to `01c38a9` if you want, maybe it's just a glitch and doesn't matter so much?

Reply:
okay, if `19db89f` is the latest, let's leave it at that. If any issues arise after merging, we can revisit.